.github/workflows/deploy-staging.yml

```
@@ -0,0 +1,48 @@
+name: Stage the app
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  build:
```

actionlint (1.7.8)

"steps" section is missing in job "build"

(syntax-check)